REQUIRE

```diff
@@ -1,7 +1,6 @@
-julia 0.6
-Compat 0.24
+julia 0.7-beta
```

Comment:
Should we just go with `julia 0.7` at this point?  I think thats encouraged after the first release version is out